### PR TITLE
DRUP-525: Add roadmap to module

### PR DESCRIPTION
**Pull request by cnovak**
* Fixed up install instructions
* Added planned features and status of project
* Added additional information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,54 @@
-Apigee Monetization Drupal module
----
+# Apigee Monetization Drupal module
 
-The Apigee Monetization module enables you to integrate Drupal 8 with Apigee Edge Monetization features.
+The Apigee Monetization module enables you to integrate Drupal 8 with Apigee Edge Monetization features. You must have
+a [monetization enabled Apigee Edge organization](https://docs.apigee.com/api-platform/monetization/enabling-monetization-organization)
```
Comment (jbennett13) [inline README.md]:
```suggestion
an Apigee Edge organization with [monetization enabled](https://docs.apigee.com/api-platform/monetization/enabling-monetization-organization)
```

File: README.md [hunk 2]
```diff
@@ -1,39 +1,54 @@
...
-**Note**: The Apigee Monetization module is dependent on the [Apigee Edge](https://www.drupal.org/project/apigee_edge) module.
+The Drupal documentation pages for this module give an [overview of the features of the Apigee monetization module](https://www.drupal.org/docs/8/modules/apigee-monetization/understand-how-app-developers-interact-with-apigee-monetization#explore-the-apigee-monetization-features-in-the-developer-portal).
+
+This module includes the following submodules:
+* **Apigee Monetization Add Credit:** adds the ability to create a Drupal Commerce "Add Credit" product to add money to 
+  a developer's prepaid balance.
+
+## Module status
+
+The core functionality of this module is complete and we are currently completing final testing and critical
```
Comment (jbennett13) [inline README.md]:
```suggestion
The core functionality of this module is complete and we are conducting final testing and critical
```

File: README.md [hunk 3]
```diff
@@ -1,39 +1,54 @@
...
 
-> The Apigee Monetization module depends on the Apigee Edge module being installed and configured to connect to the user's Apigee Edge account. 
+## Installing
 
-1. Install and enable the Apigee Edge module (see that module's README.md file for details).
-2. Go to `/admin/config/apigee-edge/settings` and create an authentication key.
-3. Install the Apigee Edge Monetization module using [Composer](https://getcomposer.org/).
+This module must be installed on a Drupal site that is managed by Composer.  Drupal.org has documentation on how to
+[use Composer to manage Drupal site dependencies](https://www.drupal.org/docs/develop/using-composer/using-composer-to-manage-drupal-site-dependencies) 
+to get you quickly started.
```
Comment (jbennett13) [inline README.md]:
```suggestion
to get you started quickly.
```

File: README.md [hunk 4]
```diff
@@ -1,39 +1,54 @@
...
+  
+1. Install the Apigee Edge Monetization module using [Composer](https://getcomposer.org/).
   Composer will download the Apigee Edge Monetization module and all its dependencies.
   **Note**: Composer must be executed at the root of your Drupal installation.
   For example:
    ```
    cd /path/to/drupal/root
    composer require drupal/apigee_m10n
    ```
+   For more information about installing contributed modules using composer, read [how to download contributed modules and themes using Composer](https://www.drupal.org/docs/develop/using-composer/using-composer-to-manage-drupal-site-dependencies#managing-contributed).
```
Comment (jbennett13) [inline README.md]:
```suggestion
   For more information about installing contributed modules using Composer, read [how to download contributed modules and themes using Composer](https://www.drupal.org/docs/develop/using-composer/using-composer-to-manage-drupal-site-dependencies#managing-contributed).
```

File: README.md [hunk 5]
```diff
@@ -1,39 +1,54 @@
...
-### Requirements
-
-* Apigee Monetization module depends on the Apigee Edge module and a number of drupal commerce modules (see apigee_m10n.info.yml)
-* **Please check [composer.json](https://github.com/apigee/apigee-edge-drupal/blob/8.x-1.x/composer.json) for required patches.** Patches prefixed with "(For testing)" are only required for running tests. Those are not necessary for using this module. Patches can be applied with the [cweagans/composer-patches](https://packagist.org/packages/cweagans/composer-patches) the plugin automatically or manually.
-
-### Development
+## Development
 
-Development is happening in our [GitHub repository](https://github.com/apigee/apigee-m10n-drupal). The drupal.org issue queue is disabled, we use the [Github issue queue](https://github.com/apigee/apigee-m10n-drupal/issues) to coordinate development.
+Development is happening in our [GitHub repository](https://github.com/apigee/apigee-m10n-drupal). The Drupal.org issue queue is disabled, we use the [Github issue queue](https://github.com/apigee/apigee-m10n-drupal/issues) to coordinate development.
```
Comment (jbennett13) [inline README.md]:
```suggestion
Development is happening in our [GitHub repository](https://github.com/apigee/apigee-m10n-drupal). The Drupal.org issue queue is disabled; we use the [Github issue queue](https://github.com/apigee/apigee-m10n-drupal/issues) to coordinate development.
```

**Review by jbennett13 — COMMENTED** [review]
Just a few tiny suggestions for your review.

File: README.md (hunk 2 above)
Comment (earth2marsh) [reply README.md]: +1, and a comma after "complete"

File: README.md [hunk 6]
```diff
@@ -1,39 +1,54 @@
...
+1. Install the Apigee Edge Monetization module using [Composer](https://getcomposer.org/).
   Composer will download the Apigee Edge Monetization module and all its dependencies.
   **Note**: Composer must be executed at the root of your Drupal installation.
   For example:
    ```
    cd /path/to/drupal/root
    composer require drupal/apigee_m10n
    ```
+   For more information about installing contributed modules using composer, read [how to download contributed modules and themes using Composer](https://www.drupal.org/docs/develop/using-composer/using-composer-to-manage-drupal-site-dependencies#managing-contributed).
+2. Click **Extend** in the Drupal administration menu.
```
Comment (earth2marsh) [inline README.md]: Do we use "click" in our docs style? Could "choose" work?

**Comment by cnovak** [comment]
Updated with @earth2marsh @jbennett13 changes

**Review by arunz6161 — APPROVED** [review]

**Review by arunz6161 — APPROVED** [review]

**Review by arunz6161 — APPROVED** [review]

**Review by arunz6161 — APPROVED** [review]

**Pull request merged**